code/game/objects/items/weapons/surgery_sutures.dm

```diff
@@ -0,0 +1,38 @@
+/obj/item/tool/suture/ //parent type, should never appear ingame
+	name = "Suture"
+	desc = "Some sort of simple surgical tool. this one doesn't seem to do anything."

```

Comment:
This (I realize it's an abstract type but come on)

Reply:
this what?
what do you want me to do?
the graft and suture not be subclasses of a suture?

Reply:
You didn't capitalize the word "This"